tests/unit/plugins/v2/test_autotools.py

```diff
@@ -53,14 +59,15 @@ def test_get_build_environment():
 def test_get_build_commands():
     class Options:
         autotools_configure_parameters = list()
+        autotools_configure_environment = list()
 
     plugin = AutotoolsPlugin(part_name="my-part", options=Options())
 
     assert plugin.get_build_commands() == [
         "[ ! -f ./configure ] && [ -f ./autogen.sh ] && env NOCONFIGURE=1 ./autogen.sh",
         "[ ! -f ./configure ] && [ -f ./bootstrap ] && env NOCONFIGURE=1 ./bootstrap",
         "[ ! -f ./configure ] && autoreconf --install",
-        "./configure",
+        "CC=${SNAPCRAFT_ARCH_TRIPLET}-gcc ./configure --host=${SNAPCRAFT_ARCH_TRIPLET}",
```

Comment:
More generically, this should perhaps be:

```
./configure --build "$(/usr/share/libtool/build-aux/config.guess)" --host "$SNAPCRAFT_ARCH_TRIPLET"
```

With that autotools should find the correct compiler, linker, etc.

Reply:
And snapcraft should export the build-machines arch triplet for convenience...  perhaps SNAPCRAFT_BUILD_ARCH_TRIPLET?

Reply:
https://www.gnu.org/software/automake/manual/html_node/Cross_002dCompilation.html